docs/units/combining_and_defining.rst

```diff
@@ -51,6 +51,8 @@ complex operations. To do this, it is recommended to use
 Defining units
 ==============
 
+.. _defining_units:
+
```

Comment:
There shouldn't be any links that are not used anywhere.
```suggestion
```

Reply:
I added it because I'm using it in the `docs/table/mixin_columns.rst` new example. Although, with a naked eye, I'm unsure as to why is it being marked as never used in the editor, but it's working in my built html files.

Reply:
It is used, I was mistaken.

Reply:
But if you want to link to a section then the link needs to be above the section title. If it's below it, like it is now, then it's actually linking to an unmarked location just below the title. You might not be able to tell the difference because this section is close to the bottom of the page, but if you make the browser window very narrow before clicking on the link you should see that the link takes you past the title.